doc/ci_admin_guide/Dependencies.md

```diff
@@ -0,0 +1,31 @@
+## How to check what is installed on builders
+
+## Prerequisites:
+1. Checkout `github.com/Juniper/contrail-windows-ci` repository.
+1. Enter `ansible/` directory.
+1. Prepare your environment by going through steps described in `README.md`. 
```

Comment:
We could add `host environment`, because i was wondering for a moment at the beginning of reading this what environment i had to prepare.